src/fondant/component_spec.py

```diff
@@ -263,6 +263,12 @@ def from_fondant_component_spec(
                     "type": "String",
                     "default": "None",
                 },
+                {
+                    "name": "cache",
+                    "description": "Set to False to disable caching, True by default.",
+                    "type": "Boolean",
+                    "default": "True",
```

Comment:
Can we not pass a bool here instead of a string?

Reply:
https://github.com/kubeflow/pipelines/issues/5111

Reply:
All default arguments are currently defined as strings even `None`. We also have do do this in the component spec. Otherwise some weird behavior will occur. In this example, setting "`True"` or `True` will still result in True but setting "`False`" and `False` will result in False and True respectively. Something about the way kubeflow evaluates arguments.